R/DataPulls.R

```diff
@@ -102,6 +102,17 @@
 #' Get preference score data from data stored in OHDSI comparative effectiveness data model,
 #' The output of this function  maybe used to create plots or tables
 #' that compares the preference score distribution between target and comparators.
+#' 
+#' example1: If the comparative effectiveness results are stored in a 
```

Comment:
Well now it is no longer an example, so drop 'example1:' and 'example2'.

Reply:
makes sense. thank you